invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/components/base.js

```diff
@@ -16,89 +20,121 @@ import {
-        <Grid.Row verticalAlign="middle">
-          <Grid.Column width={4}></Grid.Column>
-          <Grid.Column width={8} textAlign="center" floated="right">
-            <Pagination
-              options={{
-                size: "mini",
-                showFirst: false,
-                showLast: false,
-              }}
-            />
-          </Grid.Column>
-          <Grid.Column textAlign="right" width={4}>
-            <ResultsPerPage
-              values={paginationOptions.resultsPerPage}
-              label={(cmp) => (
-                <>
-                  {" "}
-                  {cmp} {i18next.t("results per page")}
-                </>
-              )}
-            />
+                    </Grid.Column>
+                    <Grid.Column
+                      width={12}
+                      textAlign="right"
+                      className="padding-r-5"
+                    >
+                      {sortOptions && (
+                        <Sort
+                          values={sortOptions}
+                          label={(cmp) => (
+                            <>
+                              {i18next.t("Sort by")} {cmp}
+                            </>
+                          )}
+                        />
+                      )}
+                    </Grid.Column>
+                  </Grid.Row>
+                </Overridable>
+                <Overridable id="DashboardResultView.resultList" {...props}>
+                  <Grid.Row>
+                    <Grid.Column>
+                      <ResultsList />
+                    </Grid.Column>
+                  </Grid.Row>
+                </Overridable>
+              </Grid>
+            </Segment>
           </Grid.Column>
         </Grid.Row>
-      </Overridable>
-    </Grid>
+        <Overridable id="DashboardResultView.resultFooter" {...props}>
+          <Grid.Row verticalAlign="middle">
+            <Grid.Column width={4}></Grid.Column>
+            <Grid.Column width={8} textAlign="center" floated="right">
+              <Pagination
+                options={{
+                  size: "mini",
+                  showFirst: false,
+                  showLast: false,
+                }}
+              />
+            </Grid.Column>
+            <Grid.Column textAlign="right" width={4}>
+              <ResultsPerPage
+                values={paginationOptions.resultsPerPage}
+                label={(cmp) => (
+                  <>
+                    {" "}
+                    {cmp} {i18next.t("results per page")}
+                  </>
+                )}
+              />
+            </Grid.Column>
+          </Grid.Row>
+        </Overridable>
+      </Grid>
+    )
   );
 }
+
+export const DashboardSearchLayoutHOC = ({
+  searchBarPlaceholder = "",
+  newBtn = () => null,
+  ...props
+}) => {
+  const DashboardUploadsSearchLayout = (props) => (
+    <Container>
+      <Grid>
+        <Grid.Row columns={3}>
```

Comment:
nit: only 2 columns in the second row

Reply:
This row has 3 columns :) Maybe I misunderstood your comment but this is applying only for the row not the whole grid!

Reply:
sorry I misread the prop :) you are right.